netbox/extras/conditions.py

```diff
@@ -132,20 +132,26 @@ class ConditionSet:
     def __init__(self, ruleset):
         if type(ruleset) is not dict:
             raise ValueError(f"Ruleset must be a dictionary, not {type(ruleset)}.")
-        if len(ruleset) != 1:
+        if len(ruleset) < 1:
             raise ValueError(f"Ruleset must have exactly one logical operator (found {len(ruleset)})")
```

Comment:
This need some adjustment (the check effectively catches empty dicts)

Reply:
I think this validation could be removed, could not?

Reply:
I see in https://github.com/netbox-community/netbox/blob/develop/netbox/extras/models/models.py there are already checks for empty conditions, so I'd just remove the `len(ruleset)` check.